internal/cmd/loglevel_value.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import "github.com/sirupsen/logrus"
+
+type LogLevel logrus.Level
```

Comment:
Not saying it's wrong, but what's the benefit of having this custom type instead of parsing and validating with `logrus.ParseLevel` directly on the root command?